themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2759,18 +2760,27 @@ If omitted, results are sorted according to their search relevance. If there is
 
 If specified more than once, the first parameter is the primary sort order and subsequent parameters control additional sorting criteria.
 
+Special consideration needs to be given if using this in tandem with the **cursor** parameter, see below.
+
 Allowed values: created, custom, delete, id, modified, module, name, package, parent.urn, pending, project, protected, provider.urn, stack, type, urn.
 
 **asc**: Whether to return results in ascending or descending sort order.
 
 **size**: How many results to return at a time.
 
+**paginate**: If requesting multiple pages of results, the pagination method to use. Should be "cursor" or "page"
+This tells the API that you would like to receive a continuation token for paging additional results.
+
 **page**: The page of results to return.
-The page parameter can only be used to fetch up 10,000 resources. If a query matches more than 10,000 resources, the "continue" parameter should be used instead.
+The page parameter can only be used to fetch up 10,000 resources. If a query matches more than 10,000 resources, the "cursor" parameter should be used instead.
 
-**continue**: A continuation token for pagination.
+**cursor**: A continuation token for pagination.
 Only available on Enterprise plans.
 
+To receive a cursor from the API, you need to issue a `paginate="cursor"` query parameter with your initial request. This informs the API that it should return a cursor as part of the JSON response.
+
+Undefined behavior can occur if the query results are impacted during pagination, for example as the result of a completed stack update. This also means that sort order must be deterministic to yield expected results, as cursor pagination respects sorting.
```

Comment:
> sort order must be deterministic to yield expected results, as cursor pagination respects sorting

I don't think I understand what this means. Cursor pagination does not respect sorting, right? I thought part of the change we made was stripping sorting from cursor based pagination to make results more stable. 

Probably we should say something simpler: 

```
Resource Search uses an eventually consistent data store and code that calls this API must account for that. For instance, stack updates trigger resources to be reindexed which may cause results to change while you are following the cursor (i.e. you may see duplicate results while following the cursor if updates are occurring frequently).
```

Reply:
Relates to [my comment](https://github.com/pulumi/pulumi-hugo/pull/3105/files#r1244195779) above